org.archimatetool.Archi.yml

```
@@ -0,0 +1,42 @@
+declare: &url https://www.archimatetool.com/downloads/archi.php?/5.0.2/Archi-Linux64-5.0.2.tgz
+declare: &sha1 cc073f0dff0932c6cf5f3204cf19bf230bcc3b0f
+
+app-id: org.archimatetool.Archi
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: Archi
+finish-args:
+  # X11 access
+  - --socket=x11
```

```suggestion
  - --socket=x11
  - --share=ipc
```

this is need for X11